pkg/util/rest/server.go

```diff
@@ -249,28 +249,28 @@ func (s *Server) list(spec *ServerObjectSpec, w http.ResponseWriter, r *http.Req
 
 // AddObjectHandlers adds the server a handlers for managing a specific object type.
 func (s *Server) AddObjectHandlers(spec *ServerObjectSpec) {
-	r := s.Router()
+	router := s.Router()
 
-	r.Route(spec.BasePath, func(r chi.Router) {
-		r.Post("/", func(w http.ResponseWriter, r *http.Request) {
+	router.Route(spec.BasePath, func(cr chi.Router) {
```

Comment:
In the original form, this included 3 (!) variables named `r`:
- the chi.Router associated with the server (renamed: `router`);
- the chi.Router variable passed as the (unnamed) function parameter (renamed: `cr`); and
- the http.Request variable passed to handlers (unchanged)